gpflow/models/sgpr.py

```diff
@@ -151,15 +150,17 @@ class SGPR_deprecated(SGPRBase_deprecated):
 
     CommonTensors = namedtuple("CommonTensors", ["A", "B", "LB", "AAT", "L"])
 
-    def maximum_log_likelihood_objective(self) -> tf.Tensor:
+    # type-ignore is because of changed method signature:
+    def maximum_log_likelihood_objective(self) -> tf.Tensor:  # type: ignore
         return self.elbo()
 
-    def _common_calculation(self):
+    def _common_calculation(self) -> "SGPR.CommonTensors":
         """
         Matrices used in log-det calculation
 
-        :return: A , B, LB, AAT with :math:`LLᵀ = Kᵤᵤ , A = L⁻¹K_{uf}/σ, AAT = AAᵀ, B = AAT+I, LBLBᵀ = B`
-        A is M x N, B is M x M, LB is M x M, AAT is M x M
+        :return: A , B, LB, AAT with :math:`LLᵀ = Kᵤᵤ , A = L⁻¹K_{uf}/σ, AAT = AAᵀ,
```

Comment:
Those superscript and subscript unicode characters inside the :math: block are giving me a headache, but that's not part of this PR!